internal/hcsoci/resources_lcow.go

```diff
@@ -6,19 +6,33 @@ package hcsoci
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"os"
 	"path"
 	"path/filepath"
 	"strings"
 
 	"github.com/Microsoft/hcsshim/internal/log"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
 )
 
 const lcowMountPathPrefix = "/mounts/m%d"
 const lcowGlobalMountPrefix = "/run/mounts/m%d"
+const lcowNvidiaMountPath = "/run/nvidia"
+
+func getNvidiaGPUVHDPath(coi *createOptionsInternal) (string, error) {
+	gpuVHDPath, ok := coi.Spec.Annotations["io.microsoft.lcow.nvidiagpuvhdpath"]
```

Comment:
We should define an annotation variable for `io.microsoft.lcow.nvidiagpuvhdpath`.

Reply:
Fixing this in my next push :)